### PR TITLE
feature/should-fall-through

### PR DESCRIPTION
**Pull request by TeoTN**
Implement #10 - optional fall through pattern matching

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +54,10 @@ const createBrowserHistory = rsr.createBrowserHistory;
 
 const history = createBrowserHistory();
 
+const options = {
+  shouldFallThrough: true,
```
Comment (jfairbank) [inline README.md]: matchAll

File: README.md [hunk 2]
```diff
@@ -103,6 +110,59 @@ your application will continue to function when you hit other routes. That also
...
+```
+
+## Options
+
+The `router` saga may also take a third argument - an `options` object - which
+allows to specify additional behaviour as described below:
+
+Key                 | Description
+--------------------|--------------------------------------------------------
+`shouldFallThrough` | Determines whether route matching should take into account all matching rules
```
Comment (jfairbank) [inline README.md]: matchAll

**Review by jfairbank — CHANGES_REQUESTED** [review]

**Comment by jfairbank** [comment]
Hi, @TeoTN. I commented on two fixes in the docs.

Also, can you bring this PR current with master and fix any merge conflicts. I appreciate that you put all the changes in #13, but I'd prefer to have separate PRs. If you can fix this PR, then I'm ready to merge :)

**Comment by TeoTN** [comment]
I have updated `README.md` and merged with master - I hope that it suits you and will be released soon :) Thanks for your help!

**Review by jfairbank — APPROVED** [review]

**Comment by jfairbank** [comment]
Thanks, @TeoTN!

I'll try to get a release out in a day or two.

**Pull request merged**